### DISCUSSION TITLE
FATAL: the database system is starting up

### DISCUSSION THREAD

**Discussion by dprophet** [mozilla/hubs#5263] [Hubs Cloud Questions]
When I go through the process hubs-cloud-aws-quick-start.html process, the Hubs service fails to properly provision. If I go into the cloudwatch and query the logs for FATAL I see that the postgres database is returning a FATAL error. This causes a full system rollback and termination.

Here is an actual screen captures.

![database_fatal_error](https://user-images.githubusercontent.com/3594022/159009728-8cd23166-a233-4dcd-a66d-9a9c2c9c4a12.png)
![database_fatal_error2](https://user-images.githubusercontent.com/3594022/159009730-5353246d-ddbd-4f97-a2ff-22cead3fbffa.png)

**Comment by pattersonbl2** [comment]
Are you trying to run a custom client?   This is the first time I have seen this error.
